File: WorldFeed/Server/src/WorldFeed.Common/Models/TwitterEntities/ExtendedEntities/VideoInformationEntity.cs
namespace WorldFeed.Common.Models.TwitterEntities.ExtendedEntities
{
    using Newtonsoft.Json;

    using global::WorldFeed.Common.Public.Models.Entities.ExtendedEntities;

    public class VideoInformationEntity : IVideoInformationEntity
    {
        [JsonProperty("aspect_ratio")]
        public int[] AspectRatio { get; set; }


        [JsonProperty("duration_millis")]
        public int DurationInMilliseconds { get; set; }


        [JsonProperty("variants")]
        public IVideoEntityVariant[] Variants { get; set; }
    }
}

File: WorldFeed/Client/src/app/core/Core/Models/TwitterEntities/ExtendedEntities/VideoInformationEntity.ts
import {IVideoInformationEntity} from "../../../../Public/Models/Entities/ExtendedEntities/IVideoInformationEntity";
import {IVideoEntityVariant} from "../../../../Public/Models/Entities/ExtendedEntities/IVideoEntityVariant";

export class VideoInformationEntity implements IVideoInformationEntity {
  // [JsonProperty("aspect_ratio")]
  public aspectRatio: number[];

  // [JsonProperty("duration_millis")]
  public durationInMilliseconds: number;

  // [JsonProperty("variants")]
  public variants: IVideoEntityVariant[];
}
